ghost/admin/app/components/gh-nav-menu/main.js

```diff
@@ -111,6 +112,11 @@ export default class Main extends Component.extend(ShortcutsMixin) {
         this.billing.openBillingWindow(this.router.currentURL);
     }
 
+    @action
+    toggleExploreWindow() {
+        this.explore.openExploreWindow(this.router.currentURL);
```

Comment:
Not directly related to this PR but I think we can drop the arguments to `openExploreWindow()` and [clean up some of the logic](https://github.com/TryGhost/Ghost/blob/main/ghost/admin/app/services/explore.js#L93-L111). The first argument passed in is used to store the `previousUrl` notionally for when the modal gets closed but this isn't a modal and doesn't have any "close modal" functionality so it's never used, same with `childRoute` as that argument is never passed in from any method calls.

Reply:
I have removed the route handling logic. We should probably really tackle that same thing for billing as well, as it also isn't using a full screen modal